app/routes.py

```diff
@@ -1,2 +1,72 @@
 from flask import Blueprint
+from app.models.task import Task
+from app import db
+from flask import request, Blueprint, make_response, jsonify
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.route("", methods=["POST","GET"])
+def handle_tasks():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return {"details": f"Invalid data"}, 400
+        
+        new_task = Task(
+            id=request_body["id"],
+            title=request_body["title"],
+            description=request_body["description"],
+            completed_at=request_body["completed_at"]
+        )
+
+        db.session.add(new_task)
+        db.session.commit()
+        
+        if new_task.completed_at == None:
+            new_task.completed_at = False
+        else:
+            new_task.completed_at = True
```

Comment:
move this above line 23 and edit it to add to the Task call. For example: 
```suggestion
        
if "completed_at" in request_body:        
             new_task.completed_at = request_body["completed_at"]
         
```